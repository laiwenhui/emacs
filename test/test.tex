% Created 2018-08-20 Mon 23:13
% Intended LaTeX compiler: pdflatex
\documentclass[11pt]{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{graphicx}
\usepackage{grffile}
\usepackage{longtable}
\usepackage{wrapfig}
\usepackage{rotating}
\usepackage[normalem]{ulem}
\usepackage{amsmath}
\usepackage{textcomp}
\usepackage{amssymb}
\usepackage{capt-of}
\usepackage{hyperref}
\usepackage{color}
\usepackage{listings}
\usepackage[UTF8]{ctex}
\setmainfont{Songti SC}
\makeatletter \def\@maketitle{\null \begin{center} {\vskip 5em \Huge \@title} \vskip 30em {\LARGE \@author} \vskip 3em {\LARGE Aug 20, 2018} \end{center} \newpage} \makeatother
\author{laiwenhui}
\date{\today}
\title{test}
\hypersetup{
 pdfauthor={laiwenhui},
 pdftitle={test},
 pdfkeywords={},
 pdfsubject={},
 pdfcreator={Emacs 26.1 (Org mode 9.1.9)}, 
 pdflang={English}}
\begin{document}

\maketitle
\setcounter{tocdepth}{2}
\tableofcontents

\#+STARTUP showall
\newpage



\definecolor{keywords}{RGB}{255,0,90}
\definecolor{comments}{RGB}{60,179,113}
\definecolor{fore}{RGB}{249,242,215}
\definecolor{back}{RGB}{51,51,51}
\lstset\{
  basicstyle=\color{fore},
  keywordstyle=\color{keywords},
  commentstyle=\color{comments},
  backgroundcolor=\color{back}
\}


\section{你好}
\label{sec:orgec4f51c}
\lstset{language=C,label= ,caption= ,captionpos=b,numbers=none}
\begin{lstlisting}

int main(int argc, char const *argv[])
{
    int retval = 0;

        /* code */
//code
        printf("hello world\n");

        return retval;
}


\end{lstlisting}
\end{document}
