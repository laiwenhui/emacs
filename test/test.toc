\contentsline {section}{\numberline {1}你好}{3}{section.1}
