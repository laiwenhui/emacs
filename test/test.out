\BOOKMARK [1][-]{section.1}{\376\377\117\140\131\175}{}% 1
